descriptors/ModuleDescriptor-template.json

```diff
@@ -120,6 +120,38 @@
             "inventory-storage.call-number-types.collection.get"
           ]
         },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/data-export/quick-export",
+          "permissionsRequired": [
+            "data-export.quick.export.post"
+          ],
+          "modulePermissions": [
+            "users.item.get",
+            "source-storage.sourceRecords.get",
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.holdings.collection.get",
+            "inventory-storage.items.collection.get",
+            "inventory-storage.nature-of-content-terms.collection.get",
+            "inventory-storage.identifier-types.collection.get",
+            "inventory-storage.contributor-name-types.collection.get",
+            "inventory-storage.locations.collection.get",
+            "inventory-storage.location-units.libraries.collection.get",
+            "inventory-storage.location-units.campuses.collection.get",
+            "inventory-storage.location-units.institutions.collection.get",
+            "inventory-storage.loan-types.collection.get",
+            "inventory-storage.material-types.collection.get",
+            "inventory-storage.instance-types.collection.get",
+            "inventory-storage.instance-formats.collection.get",
+            "inventory-storage.electronic-access-relationships.collection.get",
+            "configuration.entries.collection.get",
```

Comment:
Put this permission `"configuration.entries.collection.get"`, after `"inventory-storage.items.collection.get"`, just to do not mix it with perms for reference data, please